streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ReferenceContainer.java

```diff
@@ -30,7 +30,7 @@
     public Admin adminClient;
     public TaskManager taskManager;
     public StreamsMetadataState streamsMetadataState;
-    public final AtomicInteger assignmentErrorCode = new AtomicInteger();
+    public AtomicInteger assignmentErrorCode = new AtomicInteger();
```

Comment:
This should probably stay `final` so we don't accidentally change it ever

Reply:
I was changing it intentionally but I think I can get away with not